spec.html

```diff
@@ -5836,7 +5836,7 @@ <h1>
         1. If ! ToString(_n_) is _argument_, return _n_.
         1. Return *undefined*.
       </emu-alg>
-      <p>A <dfn variants="canonical numeric strings">canonical numeric string</dfn> is any String value for which the CanonicalNumericIndexString abstract operation does not return *undefined*.</p>
+      <p>A <dfn variants="canonical numeric strings">canonical numeric string</dfn> is any String for which the CanonicalNumericIndexString abstract operation does not return *undefined*. Each such String corresponds with a distinct Number.</p>
```

Comment:
Why call out "Each such String corresponds with a distinct Number"?

Reply:
It captures the insight provided at https://github.com/tc39/ecma262/issues/3565#issuecomment-2801128138 . Canonical numeric strings cover all potential properties referenced by `value[number]`, so use of the category rather than e.g. integer index allows an implementation such as SpiderMonkey to avoid the string conversion altogether.

Reply:
I see. I think this 1-1 sense is captured in "canonical".

If you feel strongly about capturing the insight from anba's thread, I would prefer doing so directly with a note: because canonical numeric strings are the result of ToString(n) for all Number values n, and because TypedArray treats canonical numeric string property keys specially, an implementation can treat Number values as property keys for TypedArray objects.

Reply:
Sure, added that kind of note in the TypedArray section and removed this sentence.

Reply:
Thanks.